src/Components/Login.js

```diff
@@ -39,11 +40,17 @@ const Login = () => {
 
     const formSubmitHandler = (event) => {
         event.preventDefault();
-        if (isUsernameError === false && isPasswordError === false) {
+        if (!isUsernameError && !isPasswordError) {
             setFormValidity(true);
         }
         if (formValidity === true) {
-            return //will integrate api later in this
+            Axios.post("http://localhost:3000/login", { username: enteredUsername, password: enteredPassword })
+                .then(response => {
+                    // if (!response.ok) {
+                    console.log(response);
+                    // }
+                })
+                .catch(err => alert("Error Code:" + err.response.status + " Error: " + err.response.statusText))
```

Comment:
Dont use alerts since it breaks the UI flow. If error display is not implemented just leave it as a console.error

Reply:
Replaced alert with console.log in both login.js and signup.js